row-data-gateway/README.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: pattern
+title: Row Data Gateway
+folder: row-data-gateway
+permalink: /patterns/row-data-gateway/
+categories: Structural
+tags:
+ - Data access

```

Comment:
Add `language: en` as described in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute